etl/steps/data/meadow/idmc/2024-08-02/internal_displacement.meta.yml

```diff
@@ -0,0 +1,163 @@
+      The number of internal displacements within a specific year is usually higher than the increase in the number of internally displaced people from one year to another. The number of internal displacements includes movements of newly displaced people as well as repeated and multiple movements of people already living in displacement.
+    conflict: |-
+      We track internal displacements triggered by conflict and violence [..], this includes international armed conflict, non-international armed conflict, communal violence, crime-related violence, civilian-state violence and other forms of violence. Most often, we categorize each conflict-related displacement in line with the Rule of Law in Armed Conflicts online portal by the Geneva Academy of International Humanitarian Law and Human Rights.
+    disaster: |-
+      We group movements of people triggered by disasters into two categories: weather-related and geophysical. Based on EM-DAT, the international disaster database of the Centre for Research on the Epidemiology of Disasters, we categorize displacements according to more than 20 different hazard types. Weather-related displacements, for example, include hazards such as floods, droughts, landslides, storms – including cyclones and other storms –, wildfires and extreme temperatures. Geophysical displacements include, for example, earthquakes, volcanic eruptions and landslides.
+  processing_notes:
+    population_share: |-
+      The share of internally displaced people of the total population is calculated as the number of internally displaced people in a country divided by the total population of the country, multiplied by 100. For this calculation we use our long-term population data set, which can be explored in the [population data explorer](https://ourworldindata.org/explorers/population-and-demography).
+
+
+dataset:
+  update_period_days: 365
+
+tables:
+  internal_displacement:
+    variables:
+      total_stock_displacement:
+        title: Internally displaced people
+        description_short: |-
+          [Internally displaced people](#dod:internally-displaced) due to both conflict and violence and natural disaster. This includes all people who are currently displaced, no matter which year they originally got displaced.
+        unit: people
+        short_unit: ""
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+          - {definitions.idmc_definitions.disaster}
+        description_processing: This figure is the sum of the number of people displaced due to conflict and violence and the number of people displaced due to natural disasters.
+      share_of_internally_displaced_pop:
+        title: Share of internally displaced people of total population
```

Comment:
nit - this title sounds a bit weird to me and I'd maybe rephrase it to "Internally displaced people as a share of total population"

Reply:
That's a good idea - thank you!